src/Data/List/Relation/Binary/Permutation/Propositional/Properties.agda

```diff
@@ -28,9 +30,31 @@ open import Relation.Unary using (Pred)
 open import Relation.Binary
 open import Relation.Binary.PropositionalEquality as ≡
   using (_≡_ ; refl ; cong; cong₂; _≢_; inspect)
+open import Relation.Nullary
 
 open PermutationReasoning
 
+
+------------------------------------------------------------------------
+-- Permutations of empty and singleton lists
+
+module _ {a} {A : Set a} where
```

Comment:
Any chance we could use this opportunity to make`a` and `A` variables?

Reply:
Personally I'm not convinced that that is an improvement for these kind of use cases. But I'd be happy to do it if that is what the stdlib is going for.

Reply:
I like the fact that it gets rid of the variable indentation. But no worries, I won't make you do an unnecessary refactoring that you don't agree with 😄